Scripts/Helpers/ProxyHelper.cs

```diff
@@ -102,9 +71,40 @@ public class ButtonObject
         public Transform meshRoot { get { return m_MeshRoot; } }
 
         /// <summary>
-        /// Button objects to store transform and renderer references
+        /// Affordance objects to store transform and renderer references
+        /// </summary>
+        public Affordance[] affordances { get { return m_Affordances; } }
+
+        /// <summary>
+        /// Set the visibility of the renderers associated with affordances(controls/input)
         /// </summary>
-        public ButtonObject[] buttons { get { return m_Buttons; } }
+        /// Null checking before setting, as upon EXR setup, in Awake(), m_ProxyUI is null, even though it has been assigned in the inspector
+        public bool affordanceRenderersVisible { set { if (m_ProxyUI != null) m_ProxyUI.affordancesVisible = value; } }
+
+        /// <summary>
+        /// Set the visibility of the renderers not associated with controls/input
+        /// </summary>
+        public bool bodyRenderersVisible { set { if (m_ProxyUI != null) m_ProxyUI.bodyVisible = value; } }
+
+        void Start()
+        {
+            // Setup ProxyUI
+            List<Transform> origins = new List<Transform>();
+            origins.Add(rayOrigin);
+            origins.Add(menuOrigin);
+            origins.Add(alternateMenuOrigin);
+            origins.Add(previewOrigin);
+            origins.Add(fieldGrabOrigin);
+
+            if (m_ProxyUI)
+                m_ProxyUI.Setup(m_Affordances, origins);
+
+            if (m_ProxyAnimator)
+            {
+                m_ProxyAnimator.Setup(m_Affordances);
+                this.ConnectInterfaces(m_ProxyAnimator, this.rayOrigin);
```

Comment:
Is the order important here? I'd generally opt for setting interfaces first in case Setup needs injected functionality.

Reply:
Also "this." is redundant